### PR TITLE
fix password reset

### PR DESCRIPTION
**Pull request by yccgabriel**

### CODE REVIEW

File: hackman/urls.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def robots(request: HttpRequest) -> HttpResponse:
         auth_views.PasswordResetDoneView.as_view(),
         name="password_reset_done",
     ),
-    re_path(
-        "reset/(?P<uidb64>[0-9A-Za-z_-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,20})/",  # noqa
```
Comment (ppaeps) [inline hackman/urls.py:60]: This looks like some sort of input validation.  Is it safe/wise to remove this?  Is it only flake8 that gets upset about this?

**Comment by yccgabriel** [comment]
It’s not flake8 being upset.
This line was the first change when fixing the password reset flow.
Apparently this line couldn’t match the password-reset URL, but I haven’t dig deeper to try to fix the regex matching.

IMO I could ignore this pattern matching because there’s no other similar url patterns - just route them all to the PasswordResetConfirmView for input validation.
The Django code shows that it’s checking the token validity:
https://github.com/django/django/blob/main/django/contrib/auth/views.py#L277￼
django/django/contrib/auth/views.py at main · django/django
github.com

**Comment by ppaeps** [comment]
Looking at the test emails you sent, try changing that regular expression to 

`(?P<uidb64>[0-9A-Za-z_-]+)/(?P<token>[0-9A-Za-z]{1,13}-[0-9A-Za-z]{1,32})`

(20 -> 32 at the end)

**Comment by yccgabriel** [comment]
It works.  I’ll update the PR.

**Comment by ppaeps** [comment]
Thanks for making this work!

**Pull request closed without merging**